src/GitLabApiClient/Models/Groups/Requests/ProjectsGroupQueryOptions.cs

```diff
@@ -52,5 +49,41 @@ public class ProjectsGroupQueryOptions
 		/// Limit by projects starred by the current user
 		/// </summary>
 		public bool Starred { get; set; }
+
```

Comment:
Formatting is different here? tabs  & spaces?

Reply:
Ya, I want to add a PR that setups up code formatting and will fail CI 😅
Hopefully with a `editorconfig` it should be as simple a reformatting all the code.

Though it will cause rewrite in a couple of files.

Reply:
good idea, I think we can use ```dotnet format``` to fail the build if it's not formatted according to .editorconfig? or is there a better way?

Reply:
Seems to be the best way 😆